packages/big-design/src/components/Table/Row/Row.tsx

```diff
@@ -40,35 +43,68 @@ const InternalRow = <T extends TableItem>({
   };
 
   const label = isSelected ? `Selected` : `Unselected`;
+  const [isExpanded, setIsExpanded] = useState(false);
+  const itemHasChild = Boolean(item.expandableContent);
+  const colSpan = columns.length + Number(showDragIcon) + Number(isSelectable);
+
+  const toggleExpanded = () => {
+    setIsExpanded(!isExpanded);
+  };
+
+  useEffect(() => {
+    if (isDragging && isExpanded) {
+      setIsExpanded(false);
+    }
+  }, [isDragging, isExpanded]);
 
   return (
-    <StyledTableRow isSelected={isSelected} ref={forwardedRef} isDragging={isDragging} {...rest}>
-      {showDragIcon && (
-        <DataCell>
-          <DragIndicatorIcon />
-        </DataCell>
-      )}
-      {isSelectable && (
-        <DataCell key="data-checkbox" isCheckbox={true}>
-          <Checkbox checked={isSelected} hiddenLabel label={label} onChange={onChange} />
-        </DataCell>
+    <>
+      <StyledTableRow isSelected={isSelected} ref={forwardedRef} isDragging={isDragging} {...rest}>
+        {isExpandable && (
+          <DataCell withPadding={false}>
+            {itemHasChild && (
+              <Button
+                variant="subtle"
+                onClick={toggleExpanded}
+                iconOnly={isExpanded ? <ExpandMoreIcon color="gray" /> : <ChevronRightIcon color="gray" />}
```

Comment:
I see no accessibility features added to this addition. This [codepen](https://codepen.io/aardrian/pen/wvKWVBW) has a good example, which we'll have to pick and choose from.